.github/workflows/metric.yml

```diff
@@ -51,7 +51,7 @@ jobs:
           token: ${{ secrets.METRIC_SECRET_KEY }}
 
       - name: Setup Pnpm
-        uses: pnpm/action-setup@v2
+        uses: pnpm/action-setup@v4
```

Comment:
`v4` is the latest major version of [`pnpm/action-setup`](https://github.com/pnpm/action-setup) at the time of this pull request's creation.